cmd/tracee/cmd/analyze.go

```diff
@@ -37,39 +37,26 @@ func init() {
 		"Define which signature events to load",
 	)
 
-	// TODO: decide if we want to bind this flag to viper, since we already have a similar
-	// flag in rootCmd, conflicting with each other.
-	// The same goes for the other flags (signatures-dir, rego), also in rootCmd.
-	//
-	// err := viper.BindPFlag("events", analyze.Flags().Lookup("events"))
-	// if err != nil {
-	// 	fmt.Fprintf(os.Stderr, "Error: %s\n", err)
-	// 	os.Exit(1)
-	// }
-
 	// signatures-dir
 	analyze.Flags().StringArray(
 		"signatures-dir",
 		[]string{},
 		"Directory where to search for signatures in OPA (.rego) and Go plugin (.so) formats",
 	)
-	// err = viper.BindPFlag("signatures-dir", analyze.Flags().Lookup("signatures-dir"))
-	// if err != nil {
-	// 	fmt.Fprintf(os.Stderr, "Error: %s\n", err)
-	// 	os.Exit(1)
-	// }
 
 	// rego
 	analyze.Flags().StringArray(
 		"rego",
 		[]string{},
 		"Control event rego settings",
 	)
-	// err = viper.BindPFlag("rego", analyze.Flags().Lookup("rego"))
-	// if err != nil {
-	// 	fmt.Fprintf(os.Stderr, "Error: %s\n", err)
-	// 	os.Exit(1)
-	// }
+
+	analyze.Flags().StringArrayP(
+		"log",
+		"l",
+		[]string{"info"},
+		"[debug|info|warn...]\t\tLogger options",
```

Comment:
The other flags must also be escaped with `\t` to align the column.

```
sudo ./dist/tracee analyze -h
Analyze allow you to explore signature events with past events.

...

Flags:
  -e, --events stringArray           Define which signature events to load
  -h, --help                         help for analyze
  -l, --log stringArray              [debug|info|warn...]		Logger options (default [info])
      --rego stringArray             Control event rego settings
      --signatures-dir stringArray   Directory where to search for signatures in OPA (.rego) and Go plugin (.so) formats
```

Reply:
Nice catch